java/rocksjni/portal.h

```diff
@@ -6359,6 +6360,34 @@ class TransactionDBJni : public JavaClass {
   }
 };
 
+class OccValidationPolicyJni {
+ public:
+  static jbyte toJavaOccValidationPolicy(const ROCKSDB_NAMESPACE::OccValidationPolicy policy) {
+    switch (policy) {
+      case ROCKSDB_NAMESPACE::OccValidationPolicy::kValidateSerial:
+        return 0x0;
+      case ROCKSDB_NAMESPACE::OccValidationPolicy::kValidateParallel:
```

Comment:
I wonder if it would be better to fall-through here? e.g.:

```C++
case ROCKSDB_NAMESPACE::OccValidationPolicy::kValidateParallel:
default:
  return 0x1;
```